.travis.yml

```diff
@@ -271,6 +267,31 @@ matrix:
       script:
         # Platform-specific tests currently need a pants pex, so we bootstrap here (no -b) and set -z to run the platform-specific tests.
         - ./build-support/bin/ci.sh -ez
+
+    # Rust Clippy on Linux with nightly Rust
+    - name: rust-clippy
+      os: linux
+      dist: trusty
+      sudo: required
+      stage: Test Pants
+      language: python
+      python: "2.7.13"
+      before_install:
+        - sudo apt-get install -y pkg-config fuse libfuse-dev
```

Comment:
Hm... does clippy run the tests as well? Doesn't seem like you'd need fuse otherwise.

Reply:
Compiling the fuse crate requires `libfuse-dev` to be installed... I figured might as well include `fuse` so we can run tests in the future, but happy to remove it if you'd prefer

Reply:
...oh, libfuse-dev certainly though.